presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -315,4 +351,59 @@ public void testSelectAllDatatypesParquetFile()
                         true,
                         "kot binarny".getBytes()));
     }
+
+    private static void populateDataToHiveTable(String tableName)
+    {
+        onHive().executeQuery(format("INSERT INTO %s VALUES(" +
```

Comment:
Could you do INSERT INTO x SELECT * FROM textfile, so this data only has to change in one place?
Otherwise, add a comment about changes needing to happen both in the textfile data file and here.